### ISSUE TITLE
Conflicts with the Air formatter (indent size)

### ISSUE THREAD

**Issue by eitsupi** [r-lib/lintr#2875]
lintr complains about line 5 of the file formatted by Air below, but I don't know why.

```r
func <- function() {
  if (
    isTRUE(TRUE) ||
      (isTRUE(TRUE) &&
        isTRUE(TRUE))
  ) {
    NULL
  }
}
```

Reprex:

``` r
lintr::lint(
  text = r"(
func <- function() {
  if (
    isTRUE(TRUE) ||
      (isTRUE(TRUE) &&
        isTRUE(TRUE))
  ) {
    NULL
  }
})"
)
#> <text>:6:8: style: [indentation_linter] Indentation should be 9 spaces but is 8 spaces.
#>         isTRUE(TRUE))
#>        ^~
```

<sup>Created on 2025-07-13 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

**Comment by AshesITR** [comment]
It's because of the opening parenthesis on the preceding line, which starts a hanging indent.
Without the parenthesis, the meaning would be different and the expected indentation would be the same as your snippet.

**Comment by MichaelChirico** [comment]
cc @lionel- is this something that needs to be clarified in the tidyverse guide?

**Comment by Bisaloo** [comment]
Flagging as related: https://github.com/r-lib/lintr/issues/2703
